tools/rewriter/GenCallAsm.cpp

```diff
@@ -1102,12 +1102,15 @@ std::string emit_asm_wrapper(AbiSignature sig,
     stack_alignment = (compartment_stack_space + 8) % 16;
   }
 
-  // For now, we hardcode the existence and name of the post-condition function.
-  // The name is `${target_name}_post_condition`,
-  // and we only do this for `dav1d_get_picture`.
   std::optional<std::string> target_post_condition_name = std::nullopt;
-  if (enable_dav1d_get_picture_post_condition && target_name && *target_name == "dav1d_get_picture") {
-    target_post_condition_name = *target_name + "_post_condition";
+  if (target_name && arch == Arch::X86) {
+    // TODO implement on aarch64
```

Comment:
We should make it a hard error on ARM to have any pre/post-condition functions since we don't support them yet.

Reply:
If I do that, how do I mark the tests as x86_64 only?  Otherwise the tests will just fail.

Reply:
It's fine then, you can leave it as is.